qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -353,6 +353,7 @@ def convert_play(self, shift, instruction):
         if isinstance(instruction.pulse, library.ParametricPulse):
             command_dict = {
                 'name': 'parametric_pulse',
+                'label': str(instruction.pulse.name),
```

Comment:
What if this name is `None`, I do not think the label should be included in this case, or a default name should be generated.

Reply:
I thins pulse name defaults to string `None`, but we can assign arbitrary name, i.e. `str(pulse.id())`

Reply:
updated. default pulse name is now object id.